modules/nested-accordion/widgets/nested-accordion.php

```diff
@@ -802,48 +802,60 @@ protected function content_template() {
 		?>
 		<div class="e-n-accordion" role="tablist" aria-orientation="vertical">
 			<# if ( settings['items'] ) {
-			const elementUid = view.getIDInt().toString().substr( 0, 3 ),
+			const elementUid = view.getIDInt().toString().substring( 0, 3 ),
```

Comment:
Is this change on purpose?

> The difference between substring() and substr()
The two parameters of substr() are start and length , while for substring() , they are start and end . substr() 's start index will wrap to the end of the string if it is negative, while substring() will clamp it to 0 .

Reply:
@hein-obox Yes this was purposeful, `substr` was depricated in js and replaced by `substring`